V4 rc: options.json not published to npm. Causes build failure

@evilebottnawi @sokra we are missing the `options.json` file. It looks like it's not copied over in the build / npm ignored. Blocking me from attempting to upgrade Next.js to Webpack 5 & I don't have push / publish access. Can someone who does re-publish it with its schema file :) 


<!--
  Issues are so 🔥

  If you remove or skip this template, you'll make the 🐼 sad and the mighty god
  of Github will appear and pile-drive the close button from a great height
  while making animal noises.

  👉🏽 Need support, advice, or help? Don't open an issue!
  Head to StackOverflow or https://gitter.im/webpack/webpack.
-->

- Operating System: OSX
- Node Version: 11
- NPM Version: 6
- webpack Version: 5
- webpack-dev-middleware Version: 4.0.0-rc.0

### Expected Behavior
v4 beta should work 
<!-- Remove this section if not reporting a bug or modification request. -->

### Actual Behavior
beta does not work because the schema options are missing. 
<!-- Remove this section if not reporting a bug or modification request. -->

### Code

```js
{ Error: Cannot find module './options.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Volumes/zack_dev/OSS/next.js/node_modules/webpack-dev-middleware/dist/index.js:22:39)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3) code: 'MODULE_NOT_FOUND' }

```

```js
// additional code, HEY YO remove this block if you don't need it
```
![image](https://user-images.githubusercontent.com/25274700/74918542-131eed00-537e-11ea-92d1-31cdc47a4ba2.png)

### How Do We Reproduce?

<!--
  Remove this section if not reporting a bug.

  If your webpack config is over 50 lines long, please provide a URL to a repo
  for your beefy 🍖 app that we can use to reproduce.
-->

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "lint:js": "eslint --cache src test",
     "lint": "npm-run-all -l -p \"lint:**\"",
     "prepare": "npm run build",
-    "build": "del dist && babel src -d dist",
+    "build": "del dist && babel src -d dist --copy-files",
     "release": "standard-version",
     "security": "npm audit",
     "test:only": "cross-env NODE_ENV=test jest",

